Workbooks/ADXCluster/AtResource/ResourceInsights.workbook

```diff
@@ -429,143 +1664,720 @@
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-CacheUtilization Timeline",
+                    "formatter": 9,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-CPU",
+                    "formatter": 8,
+                    "formatOptions": {},
+                    "numberFormat": {
+                      "unit": 1,
+                      "options": {
+                        "style": "decimal"
+                      }
+                    }
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-CPU Timeline",
+                    "formatter": 9,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-IngestionUtilization",
+                    "formatter": 8,
+                    "formatOptions": {},
+                    "numberFormat": {
+                      "unit": 1,
+                      "options": {
+                        "style": "decimal"
+                      }
+                    }
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-IngestionUtilization Timeline",
+                    "formatter": 9,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-InstanceCount",
+                    "formatter": 4,
+                    "formatOptions": {},
+                    "numberFormat": {
+                      "unit": 0,
+                      "options": {
+                        "style": "decimal"
+                      }
+                    }
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-InstanceCount Timeline",
+                    "formatter": 5,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "Extents (Total)",
+                    "formatter": 4,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "Extents (Total) Timeline",
+                    "formatter": 5,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "Throttled Commands (Total)",
+                    "formatter": 8,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "Throttled Commands (Total) Timeline",
+                    "formatter": 5,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.cache/redis--usedmemorypercentage",
```

Comment:
we should also clear whatever columns we don't recognize for adx, in this case the `microsoft.cache/redis` ones